extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/OkHttpVaultClient.java

```diff
@@ -252,6 +254,18 @@ public VaultInitResponse init(int secretShares, int secretThreshold) {
         return put("sys/init", body, VaultInitResponse.class);
     }
 
+    public VaultWrapResult wrap(String token, long ttl, Object object) {
+        Map<String, String> headers = new HashMap<>();
+        headers.put("X-Vault-Wrap-TTL", "" + ttl);
+        return post("sys/wrapping/wrap", token, headers, object, VaultWrapResult.class);
+    }
+
+    @Override
+    public <T> T unwrap(String wrappingToken, Class<T> resultClass) {
+        VaultUnwrapBody body = new VaultUnwrapBody(null);
```

Comment:
I just passed a null body, which is equivalent in this case

Reply:
I know but from the point of view of clean code, I am not sure this is the best way to do it. I'd advocate for creating an empty constructor to hide the details.

Reply:
what I mean is this:
```
    @Override
    public <T> T unwrap(String wrappingToken, Class<T> resultClass) {
        return post("sys/wrapping/unwrap", wrappingToken, null, resultClass);
    }
```
are you ok with this, or are you advocating this:
```
    @Override
    public <T> T unwrap(String wrappingToken, Class<T> resultClass) {
        return post("sys/wrapping/unwrap", wrappingToken, new VaultUnwrapBody(), resultClass);
    }
```

Reply:
If am not wrong this calls happens inside the same class, although maybe I'd create a `VaultUnwrapBody.EMPTY` constant representing an empty `VaultUnwrapBody` that is empty, since it is an internal call a `null` might be ok. But a public constructor that can be used outside and in several places, I think it is better to not send a null as in my opinion it looks odd.